client/src/components/Pagination/index.tsx

```diff
@@ -0,0 +1,85 @@
+import { Dispatch, SetStateAction } from 'react'
+import './Pagination.scss'
+
+interface IProps {
```

Comment:
```suggestion
type PaginationProps = {
```

다른 컴포넌트와의 통일성을 위해 요렇게 바꾸면 좋겠습니다...!